index.js

```diff
@@ -23,7 +23,7 @@ module.exports = {
       'silvermine/no-multiline-var-declarations': 'error',
       'silvermine/indent': [ 'error', 3, { 'VariableDeclaratorOffset': { 'var': 1, 'let': 1, 'const': 3 }, 'SwitchCase': 1 } ],
 
-      'comma-dangle': [ 'error', 'only-multiline' ],
+      'comma-dangle': [ 'error', 'always-multiline' ],
```

Comment:
That approach sounds good to me.

I know that in IE <= 8, a trailing comma will cause an error. I read (but have not verified myself) that trailing commas in IE 9 in "Quirks Mode" will cause an error also. I've not been able to find any indication of a mobile browser that throws an error.

Using `only-multiline` here in `index.js` lets us choose whether or not to support IE < 9, but the onus is on the developer to remember not to use a trailing commas if IE < 9 support is required.